src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -215,25 +219,28 @@ public static void ItemRender(RenderHandItemEvent.Pre event) {
         }
     }
 
+    private static final Set<ResourceKey<Level>> worldsWithoutRainOrThunder = Set.of(
+            Methods.moon,
+            Methods.moon_orbit,
+            Methods.mars,
+            Methods.mars_orbit,
+            Methods.mercury,
+            Methods.mercury_orbit,
+            Methods.venus_orbit,
+            Methods.glacio_orbit,
+            Methods.overworld_orbit
+    );
+
     @SubscribeEvent
     public static void onWorldTick(TickEvent.WorldTickEvent event) {
         if (event.phase == TickEvent.Phase.END) {
             Level world = event.world;
 
-            if (Methods.isWorld(world, Methods.moon)
-             || Methods.isWorld(world, Methods.moon_orbit)
-             || Methods.isWorld(world, Methods.mars)
-             || Methods.isWorld(world, Methods.mars_orbit)
-             || Methods.isWorld(world, Methods.mercury)
-             || Methods.isWorld(world, Methods.mercury_orbit)
-             || Methods.isWorld(world, Methods.venus_orbit)
-             || Methods.isWorld(world, Methods.glacio_orbit)
-             || Methods.isWorld(world, Methods.overworld_orbit)) {
+            if (worldsWithoutRainOrThunder.contains(world.dimension())) {
```

Comment:
Here we ask the `Set<> worldsWithoutRainOrThunder` made earlier if any of the items inside it matches the `event.world.dimension()`. This is much faster than manually checking against each world one-by-one.